### PR TITLE
Create Navbar & Footer

### PR DESCRIPTION
**Pull request by riboney**
Current preview: 
image

### CODE REVIEW

**Comment by riboney** [comment]
Gonna try to figure out how to size the buttons, height-wise and lookup proper tailwindcss classname conventions

Should be done by tomorrow

**Comment by riboney** [comment]
https://user-images.githubusercontent.com/14286113/197364640-79bc9eba-b9cd-46c0-b1f5-3b5ccb1f00c1.mp4

File: client/src/pages/Register.tsx [hunk 1]
```diff
@@ -12,7 +13,7 @@ export interface RegisterFormValues {
 function Register() {
   return (
     <>
-      <div className="card w-96 bg-base-100 shadow-xl">
+      <div className="card w-full md:w-96 rounded-none md:rounded bg-secondary md:bg-base-100 md:shadow-xl">
```
Comment (TiciaD) [inline client/src/pages/Register.tsx:16]: If you get rid of one of the `h-full` classes on the `RootLayout` component then you won't need to add all of this, it'll fix the issue with the card getting cut off on mobile landscape view

File: client/src/components/RootLayout.tsx [hunk 2]
```diff
@@ -0,0 +1,17 @@
+import { Outlet } from 'react-router-dom';
+import Footer from './Footer';
+import Navbar from './Navbar';
+
+const RootLayout = () => {
+  return (
+    <div className="flex flex-col h-full justify-between">
+      <Navbar />
+      <main className="w-screen h-full flex flex-col justify-center items-center bg-secondary">
```
Comment (TiciaD) [inline client/src/components/RootLayout.tsx:9]: You can remove the `h-full` here or on line 7 to fix the issue with the card on the `Register` page

**Review by TiciaD — CHANGES_REQUESTED** [review]
Looks great Risby! Also I forgot to mention, you don't need to stress yourself too much about getting the precise details of the mockups, I don't think it's a big deal if the shadows or rounding is not exact, I made the mockup is more of a guide than anything so I'm not super attached to any details lol. Anyway just have a few notes to save you some extra code on certain components, looks great otherwise!

File: client/src/components/RootLayout.tsx (hunk 2 above)
Comment (riboney) [reply client/src/components/RootLayout.tsx:9]: Thank you. It fixed the contents being cutoff. Although, when screen is tall enough, the background did not fill up the space between navbar and footer. I tried various different height settings, and at the end settled on using a vertical breakpoint to toggle `h-full`. Not sure if this is the ideal approach.

**Review by TiciaD — APPROVED** [review]
Looks good Risby thanks for adding those changes!

**Pull request merged**